cdc/sink/mq/mq.go

```diff
@@ -342,6 +338,7 @@ func (k *mqSink) EmitDDLEvent(ctx context.Context, ddl *model.DDLEvent) error {
 
 // Close the producer asynchronously, does not care closed successfully or not.
 func (k *mqSink) Close(ctx context.Context) error {
+	k.flushWorker.closeMsgChan()
```

Comment:
Why doesn't `k.resolvedBuffer` need to be closed?

Reply:
see: func (k *mqSink) bgFlushTs(ctx context.Context) error {